docs/README.md

```diff
@@ -0,0 +1,27 @@
+# sbt-circe-io
```

Comment:
I think sbt-http4s-org may just symlink the README. nbd, just to avoid the duplication

Reply:
Same :smile: while in active development I really don't want to create two sets of docs that need to be kept consistent with each other. After this stabilizez and more stuff is added to the plugin the website will probably contain more stuff than the README.

Reply:
```suggestion
# sbt-circe-org
```